### DISCUSSION TITLE
How do I use System.Numerics.Vector while it seems to have duplicate canidates of methods

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/fsharp#14997] [Q&A]
Type inference couldn't figure out which overload to use because there's a method that accept ```Span<byte>``` and a method that accept ```Span<'a>```
![image](https://user-images.githubusercontent.com/35312807/228535565-a1a9f52c-fcd1-4bc6-be7a-c71caba5b8ab.png)

**Comment by brianrourkeboll** [comment]
Yes, this is a more general thing where the compiler doesn't choose the more specific/concrete overload if in theory both overloads could work (this is in contrast to C#). See https://github.com/fsharp/fslang-suggestions/issues/905

**Reply by Xyncgas** [reply]
So how would I be able to use these overloads from .NET libraries without having to write glue codes in C# which is a method that calls a specific overload with a unique namespace / method name that I can use in F# to resolve the ambiguity

**Reply by Xyncgas** [reply]
The problem is really Vector<'T> (generic) has identical methods as specialized Vector<'specialized>
We can't use the generic one after .NET 5 so for example Vector<byte>.CopyTo is something you can't write in F#
